bazel/python_rules.bzl

```diff
@@ -212,6 +210,10 @@ _generate_pb2_grpc_src = rule(
             cfg = "host",
             default = Label("//external:protocol_compiler"),
         ),
+        "_grpc_library": attr.label(
+            default = Label("//src/python/grpcio/grpc:grpcio"),
```

Comment:
What material effect should I expect with this change?  For me something like:

`bazel build //src/python/grpcio/grpc:grpcio`

already works on master, and seems to work the same with this change.

Reply:
I believe this PR has been abandoned in favor of [this PR](https://github.com/grpc/grpc/pull/27275) and [its associated gRFC](https://github.com/grpc/proposal/pull/263). Whether or not this will be completed is still uncertain. @mbeards to comment on that.

The impact is spelled out in the gRFC, but to excerpt it,

> No behavior change should be observed by the user of `py_proto_library` or `py_grpc_library` unless they rely on the (removed) `plugin` attribute.

So, unless you
1. Use `py_proto_library` in your repo and
2. Use the `plugin` attribute in that rule

you will be unaffected. Based on your `bazel build` command above, it seems likely that you would in fact be unaffected.